client/my-sites/patterns/components/copy-paste-info/style.scss

```diff
@@ -9,7 +9,7 @@
 	scroll-snap-type: x proximity;
 	scroll-padding-left: 24px;
 
-	@media ( max-width: $break-wide ) {
+	@media ( max-width: $break-huge ) {
 		padding: 0 24px;
 	}
```

Comment:
```suggestion
	max-width: 1220px;
	margin: 0 auto;
	padding: 0 24px;
```

Reply:
I agree, this does feel like a smoother transition. It also makes the CSS a bit simpler which is also nice.